tests/api/test_record_api.py

```diff
@@ -147,13 +153,11 @@ def test_read_record_with_redirected_pid(client, location, minimal_record,
     assert response.json["message"] == "Moved Permanently."
 
 
-def test_read_deleted_record(client, location, minimal_record, users,
-                             es_clear):
+def test_read_deleted_record(
+    client_with_login, location, minimal_record, es_clear, admin_user
+):
```

Comment:
this test fails b/c the permission for record deletion is currently `Disabled()`, i think

Reply:
:-) It's good, it should not be possible to delete the record (or at least it's a super admin task). I would remove the test complete. We need proper tombstone handling, etc, so we'll handle a deleted record at that time.